packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -403,9 +404,10 @@ export default class SendbirdStore extends BaseStore {
     sendFile(file: File) {
         if (!file) return;
 
+        const updated_file = renameFile(file);
         this.active_chat_channel
             ?.sendFileMessage({
-                file,
+                file: updated_file,
                 fileName: file.name,
```

Comment:
won't this pass the previous file name?

Reply:
yeah.. but that was not making a change.. the issue was with the filename inside the fileobject.. but file.name can be updated if needed.